experimental/streaming/sdk/package.go

```diff
@@ -1,9 +1,24 @@
 package sdk
 
 import (
+	"go.opentelemetry.io/api/metric"
 	"go.opentelemetry.io/api/trace"
+	"go.opentelemetry.io/experimental/streaming/exporter/observer"
 )
 
+type sdk struct {
+	resources observer.EventID
+}
+
+type SDK interface {
+	trace.Tracer
+	metric.Meter
+}
+
+var _ SDK = (*sdk)(nil)
```

Comment:
People normally write this
```
var _ SDK = &sdk{}
```

Reply:
I feel like I've seen `(*T)(nil)` more often. It is the styled presented in [_Effective Go_](https://golang.org/doc/effective_go.html#blank_implements).

Reply:
You're totally right about that.  Seeing `*` and `nil` together somewhat hurts my brain.  It's unclear why that's a preferred style over `&sdk{}`.